.github/workflows/nightly-bnb.yml

```
@@ -82,6 +82,12 @@ jobs:
           pip install -e . --no-deps
           pip install pytest-reportlog pytest-cov parameterized datasets scipy einops
           mkdir transformers-clone && git clone https://github.com/huggingface/transformers.git transformers-clone
+          if [ "${{ matrix.docker-image-name }}" = "huggingface/peft-gpu-bnb-latest:latest" ]; then
```

```suggestion
          if [ "${{ matrix.docker-image-name }}" == "huggingface/peft-gpu-bnb-latest:latest" ]; then
```
reference: https://github.com/huggingface/diffusers/blob/main/.github/workflows/pr_test_peft_backend.yml#L48